python/lsst/ts/observatory/control/auxtel/atcs.py

```diff
@@ -0,0 +1,1836 @@
+            `astropy.coordinates.Angle`
+        el : `float` or `str`
+            Target Elevation (degree). Accepts float (deg), sexagesimal string
+            (DD:MM:SS.S or DD MM SS.S) coordinates or
+            `astropy.coordinates.Angle`
+        rot_tel : `float` or `str`
+            Specify rotator angle in mount physical coordinates.
+            Accepts float (deg), sexagesimal string
+            (DD:MM:SS.S or DD MM SS.S) coordinates or
+            `astropy.coordinates.Angle`
+        target_name : `str`
+            Name of the position.
+        wait_dome : `bool`
+            Wait for dome to be in sync with the telescope? If preparing to
+            take a flat, for instance, the dome will never be in sync.
+        slew_timeout : `float`
+            Timeout for the slew command (second).
+        """
+        self.rem.atptg.cmd_azElTarget.set(
+            targetName=target_name,
+            azDegs=Angle(az, unit=u.deg).deg,
+            elDegs=Angle(el, unit=u.deg).deg,
+            rotPA=Angle(rot_tel, unit=u.deg).deg,
+        )
+        check_atdome = self.check.atdome
+        self.check.atdome = wait_dome
+        check_atdometrajectory = self.check.atdometrajectory
+        self.check.atdometrajectory = wait_dome
+
+        try:
+            await self._slew_to(
+                self.rem.atptg.cmd_azElTarget, slew_timeout=slew_timeout
+            )
+        except Exception as e:
+            self.check.atdome = check_atdome
+            self.check.atdometrajectory = check_atdometrajectory
+            raise e
+
+    async def start_tracking(self):
+        """Start tracking the current position of the telescope.
+
+        Method returns once telescope and dome are in sync.
+        """
+        raise NotImplementedError("Start tracking not implemented yet.")
+
+    async def slew_object(
+        self, name, rot_sky=0.0, rot_par=None, rot_tel=None, slew_timeout=240.0,
+    ):
+        """Slew to an object name.
+
+        Use simbad to resolve the name and get coordinates.
+
+        Parameters
+        ----------
+        name : `str`
+            Target name.
+        rot_sky : `float`, `str` or `astropy.coordinates.Angle`
+            Specify rotation in sky coordinates, clock-wise, relative to the
+            north celestial pole. Accepts float (deg), sexagesimal string
+            (DD:MM:SS.S or DD MM SS.S) coordinates or
+            `astropy.coordinates.Angle`
+        rot_par :  `float`, `str` or `astropy.coordinates.Angle`
+            Specify rotation with respect to the parallactic angle.
+            Accepts float (deg), sexagesimal string (DD:MM:SS.S or DD MM SS.S)
+            coordinates or `astropy.coordinates.Angle`
+        rot_tel : `float`, `str` or `astropy.coordinates.Angle`
+            Specify rotation in mount physical coordinates.
+            Accepts float (deg), sexagesimal string
+            (DD:MM:SS.S or DD MM SS.S) coordinates or
+            `astropy.coordinates.Angle`
```

Comment:
Apparently `rot_tel` does not actually keep the rotator at the specified physical angle. So what does it do instead? I assume it must make the instrument track the sky or the parallactic angle, but which and how would the user know? (It would certainly be easier to explain if it just kept the rotator at a fixed angle!).

I am curious what the use case is for such a strange angle, and why a user would only want to track the sky or parallactic angle after starting at the specified physical angle.

This confusion applies everywhere you have the `rot_tel` argument. Sorry to ask for more documentation. I suppose you could explain in one place and refer users to that place in other argument docs.

Reply:
I suggest more docs and renaming it to "rot_skyphys" or something -- a word with sky in it *and* tel or phys. It's a weird thing to do and it can afford to have a longer, weird name, so a reader has a clue what it means.